src/azul/project/hca/transformers.py

```diff
@@ -14,18 +14,54 @@
 
 def _project_dict(bundle: api.Bundle) -> dict:
     project: api.Project
+    laboratories: List[str]
+    contributors: List[str]
+
     project, *additional_projects = bundle.projects.values()
     reject(additional_projects, "Azul can currently only handle a single project per bundle")
+
+    laboratories = set()
+    contributors = set()
```

Comment:
I think we should call this institutions because contributors is ambiguous especially when `project.contributors` is something else.